website/docs/content/writing-guidelines/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Writing Guidelines
+caption: Guidelines for writing consistent UI copy.
+description: Guidelines for writing consistent UI copy.
+previewImage: assets/illustrations/content/writing-guidelines.jpg
+---
+
+## Date and time
+
+Present dates and times in a clear and consistent format to enhance usability across products.
```

Comment:
[super nit] This uses more plain language since "present" can mean more than one thing.
```suggestion
Display dates and times clearly and consistently to enhance usability across products.
```

> enhance usability across products

How does it enhance usability? Is there anything else we could say here that would be helpful?

Reply:
I think we can stop at the "consistently" part.